src/applications/discharge-wizard/components/gpSteps/StepTwo.jsx

```diff
@@ -35,14 +32,13 @@ const renderMedicalRecordInfo = formValues => {
         <ul>
           <li>
             You can request your <strong>VA medical records</strong> by
-            submitting VA Form 10-5345 to your local VA Medical Center.{' '}
-            <a
+            submitting VA Form 10-5345 to your local VA Medical Center.
+            <br />
+            <va-link
               download
-              rel="noopener noreferrer"
-              href="https://www.va.gov/vaforms/medical/pdf/VHA%20Form%2010-5345%20Fill-revision.pdf"
-            >
-              Download VA Form 10-5345.
-            </a>
+              text="Download VA Form 10-5345"
+              href="https://www.va.gov/find-forms/about-form-10-5345/"
```

Comment:
The previous link 404'd. Got the updated link from Danielle Thierry here: https://dsva.slack.com/archives/C01K37HRUAH/p1709739803568179